Analysis/Tasks/PWGHF/qaTaskEfficiency.cxx

```diff
@@ -151,20 +137,34 @@ struct QATrackingEfficiencyPt {
     }
   }
 
-  void process(const o2::soa::Join<o2::aod::Tracks, o2::aod::McTrackLabels>& tracks,
+  void process(const o2::soa::Join<o2::aod::Collisions, o2::aod::McCollisionLabels>& collisions,
+               const o2::soa::Join<o2::aod::Tracks, o2::aod::McTrackLabels>& tracks,
+               const o2::aod::McCollisions& mcCollisions,
                const o2::aod::McParticles& mcParticles)
   {
+    std::vector<int64_t> recoEvt(collisions.size());
```

Comment:
I agree, depending on the number of collisions it might be faster to allocate the memory once. But indeed I can resize to the number of events accepted so that we avoid the 0s.

Reply:
Was this implemented using `recoEvt.resize(nevts);`?